example/examplepb/example.proto

```diff
@@ -23,19 +23,20 @@ message User {
 	repeated Parent parents = 6;
 
 	external.ExternalUser external_user = 7;
+
 }
 
 message Address {
 	string country = 1;
-	string state = 2 [(atlas_validate.field).read_only = true];
+	string state = 2 [(atlas_validate.field) = {deny:[update, create]}];
```

Comment:
just from aesthetic point of view, can we say /does proto file syntax allows us to say/ [(atlas_validate.field).deny = [update, create]]?

Reply:
I don't know why but proto syntax isn't support this syntax